src/app/lib/config/services/pidgin.js

```diff
@@ -6,6 +6,7 @@ const pidgin = {
   locale: 'pcm',
   service: 'pidgin',
   serviceName: 'Pidgin',
+  home: 'Home',
```

Comment:
This files getting a bit messy, granted, but can `home` go inside the `translations` object please?